packages/table-dev-app/src/mutableTable.tsx

```diff
@@ -265,6 +266,9 @@ export interface IMutableTableState {
     showRowHeadersLoading?: boolean;
     showTableInteractionBar?: boolean;
     showZebraStriping?: boolean;
+    scrollDirection?: "UP" | "DOWN";
+    animationRequestId?: number;
+    previousTime?: number;
```

Comment:
nit: these don't need to live in component state, because they don't affect the `render()` code path. they can just be private class properties instead.

Reply:
mind blown! thanks for the info!